src/components/SpatialViewer/SpatialViewerContainer.js

```diff
@@ -1,39 +0,0 @@
-import {connect} from "react-redux";
-import SpatialViewer from "./SpatialViewer";

```

Comment:
Removing spatial viewer as it isn't needed for the repository, and the vitessce package has a number of vulnerabilities associated with it.